packages/wonder-blocks-banner/src/components/banner.tsx

```diff
@@ -274,12 +274,7 @@ const Banner = (props: Props): React.ReactElement => {
 
     return (
         <View
-            style={[
-                styles.containerOuter,
-                layout === "floating" && styles.floatingLayout,
-                layout === "full-width" && styles.fullWidthLayout,
```

Comment:
Here is where we stop using the layout prop. We set the border radius in the `containerOuter` styles now